src/shortcuts/deleteEmptyThoughtOrOutdent.js

```diff
@@ -60,6 +63,33 @@ const exec = (dispatch, getState) => {
   dispatch({ type: 'deleteEmptyThought' })
 }
 
+// eslint-disable-next-line jsdoc/require-jsdoc
+const canExecuteOutdent = getState => {
```

Comment:
JSDOC should only be disabled in rare cases, such as for extremely trivial functions or canonical functions like `mapStateToProps`.